clang/lib/3C/ArrayBoundsInferenceConsumer.cpp

```diff
@@ -533,7 +533,8 @@ bool GlobalABVisitor::VisitFunctionDecl(FunctionDecl *FD) {
         }
       }
 
-      for (auto &CurrNtArr : ParamNtArrays) {
+      // Do not use heuristics for param nt arrays.
+      /*for (auto &CurrNtArr : ParamNtArrays) {
         unsigned PIdx = CurrNtArr.first;
```

Comment:
Is this a change we want? It seems separate from the issue being addressed.

Reply:
Yes, this is what we want. This was not removed before.